utils/reconciler.go

```diff
@@ -106,6 +106,8 @@ func (r *ReconcilerBase) SetDiscoveryClient(discovery discovery.DiscoveryInterfa
 }
 
 var log = logf.Log.WithName("utils")
+// Level 1 corresponds to 'DEBUG'
+var logv1 = log.V(1)
```

Comment:
Assuming this log level can be set at Operator runtime, we can support users providing the log level in the Operator config map as a next step.